client/src/components/SelectionDialog/selectionTypes.ts

```diff
@@ -6,21 +6,19 @@ export const SELECTION_STATES = {
 
 export type SelectionState = (typeof SELECTION_STATES)[keyof typeof SELECTION_STATES];
 
-export interface FieldEntry {
-    key: string;
-    label?: string;
-    sortable?: boolean;
-}
-
 export interface SelectionItem {
     id: string;
     label: string;
-    details: string;
+    time?: Date | string;
+    update_time: Date | string;
```

Comment:
I've seen this can be an empty string in some cases, can we instead make it optional and change it to `updateTime?: Date` so those empty strings are undefined. In general, we should have an "adapter" or mapping function that will translate the raw item to a `SelectionItem` so we can do the conversion/unification there. The same for `time` that can be maybe also called `creationTime` for consistency.